riocli/project/whoami.py

```diff
@@ -0,0 +1,101 @@
+# Copyright 2024 Rapyuta Robotics
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import click
+from click_help_colors import HelpColorsCommand
+
+from riocli.config import Configuration
+from riocli.constants import Colors
+from riocli.exceptions import LoggedOut
+from riocli.project.util import name_to_guid
+
+
+@click.command(
+    'whoami',
+    cls=HelpColorsCommand,
+    help_headers_color=Colors.YELLOW,
+    help_options_color=Colors.GREEN,
+)
+@click.option('-p', '--project-name', type=str)
+@name_to_guid
+@click.pass_context
+def whoami(ctx: click.Context, project_name: str, project_guid: str) -> None:
+    """
+    Find the role of the current user in a project.
+    """
+    if not ctx.obj.data.get('email_id'):
+        raise LoggedOut
+
+    try:
+        role = find_role(
+            Configuration(),
```

Comment:
suggestion: Since we have context, can we pass configuration from Context itself?